src/components/KeyboardAvoidingView/index.js

```diff
@@ -3,11 +3,15 @@
  */
 import React from 'react';
 import {View} from 'react-native';
+import _ from 'underscore';
 
-const KeyboardAvoidingView = props => (
+const KeyboardAvoidingView = (props) => {
+    const viewProps = _.omit(props, ['behavior']);
```

Comment:
@0xmiroslav We should omit the other [KAV props](https://reactnative.dev/docs/keyboardavoidingview#props) too: `contentContainerStyle `, `enabled`, `keyboardVerticalOffset`